docs/feature/search/vector/index.md

```diff
@@ -121,10 +122,12 @@ VALUES
 **DQL**
 
 ```sql
-SELECT text, _score
+SELECT
+  text,
+  VECTOR_SIMILARITY(embedding, [0.3, 0.6, 0.0, 0.9]) AS score
 FROM word_embeddings
-WHERE KNN_MATCH(embedding,[0.3, 0.6, 0.0, 0.9], 2)
-ORDER BY _score DESC;
+WHERE KNN_MATCH(embedding, [0.3, 0.6, 0.0, 0.9], 2)
+ORDER BY score DESC;
 ```
```

Comment:
Also a parameterized query should work

```json
{
  "stmt":"SELECT  text, VECTOR_SIMILARITY(embedding, $1) AS score
FROM word_embeddings WHERE KNN_MATCH(embedding, $1, 2)
ORDER BY score DESC;",
  "args":[[0.3, 0.6, 0.0, 0.9]]
}
```

Reply:
Thanks a stack. I used the previous plain-SQL variant, because it's the right choice in this context.

```sql
WITH param AS
  (SELECT [0.3, 0.6, 0.0, 0.9] AS vector)
SELECT
  text,
  VECTOR_SIMILARITY(
    embedding, (SELECT vector FROM param)) AS score
FROM word_embeddings
WHERE KNN_MATCH(
  embedding, (SELECT vector FROM param), 2)
ORDER BY score DESC;
```

While it's more characters, I also used an explicit name for the synthetic column added by the `param` CTE, just calling it `vector`. We could shorten that to `iv`, as a short-hand to "input vector".

wdyt?

Reply:
We don't necessarily need to keep this layout, but as long as we are using it, it would be cool to compress the width a bit further, so I will use `sv` instead.

![image](https://github.com/user-attachments/assets/01784f4f-5323-43cd-983a-284e0439f978)

Reply:
I don't like the style too much yet, but c'est la vie. It's SQL, and it fits into the box, so we are satisfied. Thanks again.

![image](https://github.com/user-attachments/assets/80904592-a604-4865-a808-15d86730b988)

Reply:
Now that I really don't like form and style, 0cccdca895918 is trying to change the page layout a bit further, introducing asymmetry into the "Synopsis" section, in order to better accomodate for wide vs. narrow SQL code elements.